src/utils/prepareDate.ts

```diff
@@ -0,0 +1 @@
+export const prepareDate = (value:string):string => new Date(value).getFullYear().toString()
```

Comment:
ts уже знает, что будет возвращена строка
<img width="798" alt="Screen Shot 2023-01-10 at 7 01 54 PM" src="https://user-images.githubusercontent.com/83491858/211586044-d6356d30-b854-4986-9984-32a808ce65c4.png">
```suggestion
export const prepareDate = (value:string) => new Date(value).getFullYear().toString()
```

посмотри похожие кейсы
Если не требуется явного указания, лучше убрать возвращаемый тип